src/PhpSpec/Console/ContainerAssembler.php

```diff
@@ -390,8 +391,26 @@ private function setupLocator(IndexedServiceContainer $container)
         });
 
         $container->addConfigurator(function (IndexedServiceContainer $c) {
-            $suites = $c->getParam('suites', array('main' => ''));
+            $suites = [];
+            $namespaceProviderFactory = new Factory();
+            foreach ($c->getParam('namespace_providers', []) as $namespaceProviderId => $namespaceProviderConfig) {
+                $namespaceProvider = $namespaceProviderFactory->getProvider(
+                    $namespaceProviderId,
+                    $namespaceProviderConfig
+                );
+                foreach ($namespaceProvider->getNamespaces() as $namespace => $location) {
+                    $suites[md5($namespace)] =  [
```

Comment:
That `md5` is ugly, any recommendation for a slugification method I should use to generate a decent name for the suites?